templates/vue-demo-store/app/router.options.ts

```diff
@@ -0,0 +1,29 @@
+import type { RouterOptions } from "@nuxt/schema";
+
+export default <RouterOptions>{
+  scrollBehavior(to, from, savedPosition) {
+    /**
+     * Do not scroll to top when switching between product variants
+     *
+     * Assumptions:
+     * - product with variants contains minimum 2 subdirectories
```

Comment:
we should add some more info (docs or here or both?) that the format of URL is not fixed - can be configured and then, the condition won't work. 
![image](https://github.com/user-attachments/assets/a54650b8-f015-4ce2-ba99-dfcd9ee09102)